associated-token-account/program/src/lib.rs

```diff
@@ -106,7 +106,7 @@ pub fn create_associated_token_account(
             AccountMeta::new_readonly(*wallet_address, false),
             AccountMeta::new_readonly(*token_mint_address, false),
             AccountMeta::new_readonly(solana_program::system_program::id(), false),
-            AccountMeta::new_readonly(spl_token::id(), false),
+            AccountMeta::new_readonly(spl_token_2022::id(), false),
```

Comment:
same here, I think this ought to remain as spl_token.   Also this is a deprecated method, so another reason to leave it be.

Reply:
same here